### DISCUSSION TITLE
Type '(number | boolean)[]' provides no match for the signature '(err: Error, result: QueryResult<any>): void' for @types/pg

### DISCUSSION THREAD

**Discussion by sagnikc395** [DefinitelyTyped/DefinitelyTyped#69225] [Issues with a @types package]
I am using @types/pg in my project. While using tsc -b as a build script to build out the javascript code , I am getting a issue as:
<img width="1653" alt="Screenshot 2024-04-01 at 6 38 03 AM" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/118270727/9a1f3d4d-21ad-44d6-964d-e430cbb75166">

No overload matches are present.

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Will be fixed by PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69053
